control/control.go

```diff
@@ -822,7 +893,7 @@ func (p *pluginControl) SubscribeDeps(taskID string, mts []core.Metric, plugins
 				serrs = append(serrs, serror.New(err))
 				return serrs
 			}
-			pool.Subscribe(taskID, strategy.UnboundSubscriptionType)
+			pool.Subscribe(taskID)
```

Comment:
Subscribe doesn't contain the subscription type? I noticed in the change log it mentions removing them for simplicity but what is the change in behavior with them removed and how is that accomplished?